latest/index.bs

```diff
@@ -148,39 +133,47 @@ A well group SHOULD NOT be present if there are no images in the well.
 
 
 <pre>
-.                             # Root folder, potentially in S3,
+.
 │
-└── 5966.zarr                 # One plate (id=5966) converted to Zarr
-    ├── .zgroup
-    ├── .zattrs               # Implements "plate" specification
+└── 5966.zarr                 # One OME-Zarr plate (id=5966)
+    ├── zarr.json             # Implements "plate" specification
     ├── A                     # First row of the plate
-    │   ├── .zgroup
+    │   ├── zarr.json
```

Comment:
Does this `zarr.json` need to be present? I'm not entirely clear from reading the Zarr v3 spec whether you are allowed to have empty directories? (or if the rules are different from Zarr v2 with .zgroup)?

If you aren't allowed empty directories, then does this need a change/clarification to the labels section above where we have:

```
Intermediate folders are permitted but not necessary and currently contain no extra metadata
```
Do we need `zarr.json` to be shown within the `original` directory?

Reply:
There is an [ongoing discussion](https://github.com/zarr-developers/zarr-specs/issues/291) about "implicit" groups. I think the community is leaning towards disallowing these, i.e. requiring `zarr.json` files for intermediate folders.